content/en/opentelemetry/instrument/api_support/python/metrics.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Python OpenTelemetry Metrics API Support
+code_lang: metrics
+type: multi-code-lang
+code_lang_weight: 1
+further_reading:
+    - link: opentelemetry/correlate/metrics_and_traces
+      tag: Documentation
+      text: Correlate OpenTelemetry Traces and Metrics
+---
+
+## Overview
+
+{{% otel-overview-exporter lang="Python" signal="Metrics" sdk_name="dd-trace-py" %}}
+
+## Prerequisites
+
+- **Datadog SDK**: dd-trace-py version 3.18.0 or later.
+- **OpenTelemetry SDK**: The standard opentelemetry-sdk package for Python.
```

Comment:
We have steps in the setup for installing the SDK and Exporter, so it could be confusing to have them as prerequisites as well?